common/message.go

```diff
@@ -60,3 +60,11 @@ type ConnectionAuthMethod struct {
 	Type   string `json:"type"`
 	Issuer string `json:"issuer"`
 }
+
+type ModuleIdentity struct {
```

Comment:
For every public element, it's a best practice to add a comment to explain what it does

Reply:
Done.